src/components/calculator/Calculator.jsx

```diff
@@ -0,0 +1,78 @@
+import './calculator.scss';
+
+const Calculator = () => (
```

Comment:
- [ ] To fulfill one of the requirements of this project, let's make this component class-based.

Reply:
Hi @FrederickMih, ESLint doesn't let me change this functional component to a class component. What should I do?

<img width="799" alt="Screen Shot 2022-04-19 at 2 07 56 PM" src="https://user-images.githubusercontent.com/45613824/164077642-a92693b0-0a6a-4f5d-86c4-e56fe9f0e028.png">

Reply:
@FrederickMih, by the way. I also get the same linter issue when trying to convert App.js to a class-based module. I will have to convert them to functional components.

<img width="756" alt="Screen Shot 2022-04-19 at 2 28 00 PM" src="https://user-images.githubusercontent.com/45613824/164081225-99bdb266-da1f-45f9-b933-aaa1d90c1708.png">

Reply:
Hi @hsztan. Please let's try writing the component this way.

```
class Calculator extends React.PureComponent {
  render() {
    return <div>{this.props.Calculator }</div>;
  }
}
```

Thanks :pray:

Reply:
@FrederickMih, was able to create the class-based components by extending the class from `React.PureComponent` 💪

Reply:
@hsztan, that's great :+1: 

Wish you all the best :pray: